downstream/modules/aap-on-azure/con-aap-azure-architecture.adoc

```diff
@@ -0,0 +1,22 @@
+[id="con-aap-azure-architecture"]
+
+= Application Architecture
+
+{AAPonAzureName} is installed as a managed application.
+Red Hat manages both the underlying Azure resources and the software running on it while that infrastructure runs in your Azure tenant.
+
+The managed application  resource group is completely separate from other resource groups in your tenant.
+Red Hat only has access to the managed application resource group, with no visibility into other tenant resources.
+
+For information about how this works and how resources and access are isolated from the rest of your Azure resources, refer to link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/managed-applications/overview[Azure managed applications overview] in the Microsoft Azure documentation.
+
+{AAPonAzureNameShort} uses two resource groups:
+
+* A resource group in your tenant. This resource group includes a single resource referring to the {AAPonAzureNameShort} managed app deployment. 
+Red Hat has access to this managed app to perform support, maintenance, and upgrades.
+* A multi-tenant managed resource group that contains all the infrastructure needed to operate {AAPonAzureNameShort}. This multi-tenant resource group is shared between the Red Hat tenant and your tenant. Red Hat has full administrative control and you have read-only access to the resource group.
+
+The following diagram outlines the application resources and architecture that are deployed into the managed application resource group when {AAPonAzureNameShort} is deployed into your Azure subscription.
```

Comment:
Should it say `"...deployed into your Azure environment."` instead of `"...deployed into your Azure subscription."` ?

Reply:
It sounds odd, but subscription in this context means a container in your Azure instance that represents the subscription to AAP.

Reply:
@ariordan-redhat Really? Okay, then. Pls disregard the comment.